mapgen/iwyu-mapgen-macos.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+##===--- iwyu-mapgen-macos.py ---------------------------------------------===##
```

Comment:
Sorry for thinking about naming so late, but I think this should be `iwyu-mapgen-apple-libc.py` -- the Apple Libc seems to be used across macOS, iOS, tvOS and watchOS.

Reply:
You're right, that definitely is more accurate.